packages/react/src/components/card-link/card-link.tsx

```diff
@@ -15,11 +15,12 @@ const StyledIcon = styled(Icon)`
 `;
 
 const StyledLink = styled(Link)`
-    background-color: ${({ theme }) => theme.component['card-link-background-color']};
-    border: 1px solid ${({ theme }) => theme.component['card-link-border-color']};
+    background-color: ${({ theme }) => theme.component['card-background-color']};
+    border: 1px solid ${(props) => props.theme.component['card-link-border-color']};
```

Comment:
Gardons les meme syntaxes

```suggestion
    border: 1px solid ${({ theme }) => theme.component['card-link-border-color']};
```